test/saithrift/tests/saiacl.py

```diff
@@ -1634,3 +1634,453 @@ def runTest(self):
+            attr_value = sai_thrift_attribute_value_t(oid=SAI_NULL_OBJECT_ID)
+            attr = sai_thrift_attribute_t(id=SAI_VLAN_ATTR_INGRESS_ACL, value=attr_value)
+            self.client.sai_thrift_set_vlan_attribute(vlan_oid, attr)
+            # unbind this ACL table from port1s object id
+            attr_value = sai_thrift_attribute_value_t(oid=SAI_NULL_OBJECT_ID)
+            attr = sai_thrift_attribute_t(id=SAI_LAG_ATTR_INGRESS_ACL, value=attr_value)
+            self.client.sai_thrift_set_lag_attribute(lag_id1, attr)
+
+            print "Verify the traffic after ACL is removed: Expecting packets to go through"
+            print "Sending packets from port 1 to port 2"
+            send_packet(self, 0, str(pkt))
+            print "Verifying packet"
+            verify_packets(self, exp_pkt, [1])
+            print "Packet received as expected"
+
+            print "Sending packets from port 2 to port 1"
+            send_packet(self, 1, str(pkt1))
+            print "Verifying packet"
+            verify_packets(self, exp_pkt1, [0])
+            print "Packet received as expected"
+
+        finally:
+            # cleanup ACL
+            self.client.sai_thrift_remove_acl_table_group_member(acl_table_group_member_id1)
+            self.client.sai_thrift_remove_acl_table_group_member(acl_table_group_member_id2)
+            self.client.sai_thrift_remove_acl_table_group_member(acl_table_group_member_id3)
+            self.client.sai_thrift_remove_acl_table_group_member(acl_table_group_member_id4)
+            self.client.sai_thrift_remove_acl_entry(acl_entry_id1)
+            self.client.sai_thrift_remove_acl_entry(acl_entry_id2)
+            self.client.sai_thrift_remove_acl_entry(acl_entry_id3)
+            self.client.sai_thrift_remove_acl_entry(acl_entry_id4)
+            self.client.sai_thrift_remove_acl_table(acl_table_id1)
+            self.client.sai_thrift_remove_acl_table(acl_table_id2)
+            self.client.sai_thrift_remove_acl_table(acl_table_id3)
+            self.client.sai_thrift_remove_acl_table(acl_table_id4)
+            self.client.sai_thrift_remove_acl_table_group(acl_table_group_id1)
+            self.client.sai_thrift_remove_acl_table_group(acl_table_group_id2)
+
+            sai_thrift_delete_fdb(self.client, vlan_oid, dmac2, port2)
+
+            # Cleanup router config
+            sai_thrift_remove_route(self.client, vr_id, addr_family, ip_addr_subnet1, ip_mask, nhop1)
+            sai_thrift_remove_route(self.client, vr_id, addr_family, ip_addr_subnet2, ip_mask, nhop2)
+            self.client.sai_thrift_remove_next_hop(nhop1)
+            self.client.sai_thrift_remove_next_hop(nhop2)
+            sai_thrift_remove_neighbor(self.client, addr_family, rif_id1, ip_addr1, dmac1)
+            sai_thrift_remove_neighbor(self.client, addr_family, rif_id2, ip_addr2, dmac2)
+            sai_thrift_remove_lag_member(self.client, lag_member_id1)
+            self.client.sai_thrift_remove_lag(lag_id1)
```

Comment:
Need to remove rif_id1 before removing lag_id1, since rif_id1 is a port rif on lag_id1

Reply:
Addressed review comments